packages/@uppy/companion/src/standalone/helper.js

```diff
@@ -70,6 +69,9 @@ const getConfigFromEnv = () => {
         key: process.env.COMPANION_UNSPLASH_KEY,
         secret: process.env.COMPANION_UNSPLASH_SECRET,
       },
+      s3: {
+        getKey: (req, filename) => `${crypto.randomUUID()}-${filename}`,
+      },
```

Comment:
You are creating a `s3` key again which I removed in #3781. This should be moved down.

Reply:
Ah right, I couldn't make sense of why the code on `main` was doing that, thanks.